### DISCUSSION TITLE
downloaded files verus shown downloadbale amout huge difference

### DISCUSSION THREAD

**Discussion by bloke90** [TumblThreeApp/TumblThree#284] [Q&A (Question & Answer)]
I noticed recently that there is a huge difference between the shown available "files" and the onces which are dowloaded at the end

37700 "likes" to be downloaded but 980 only done
 522 likes to be downloaded but only 191 done

?

**Comment by thomas694** [comment]
Do you mean the two columns in the blog list?
Normally it indicates plenty duplicates/reblogs in a blog. Or, if you enabled the setting "global duplicate check", you already have downloaded these posts/files with another blog. In both cases, you should also see a high number under "duplicates found" in the blog details.
